lib/token.js

```
@@ -0,0 +1,221 @@
+     *   pageToken = response.nextPageToken || undefined;
+     * } while (pageToken);
+     *
+     * @see {@link https://docs.scalekit.com/apis/#tag/tokens | List Tokens API}
+     */
+    listTokens(organizationId, options) {
+        return __awaiter(this, void 0, void 0, function* () {
+            if (!organizationId) {
+                throw new Error("organizationId is required");
+            }
+            return this.coreClient.connectExec(this.client.listTokens, Object.assign(Object.assign(Object.assign({ organizationId }, ((options === null || options === void 0 ? void 0 : options.userId) && { userId: options.userId })), ((options === null || options === void 0 ? void 0 : options.pageSize) && { pageSize: options.pageSize })), ((options === null || options === void 0 ? void 0 : options.pageToken) && { pageToken: options.pageToken })));
+        });
```

Biome (2.4.4)

[error] This generator function doesn't contain yield.

(lint/correctness/useYield)